src/views/StackView/StackViewLayout.js

```diff
@@ -433,11 +445,22 @@ class StackViewLayout extends React.Component {
       screenInterpolator &&
       screenInterpolator({ ...this.props.transitionProps, scene });
 
+    // If this screen has "header" set to `null` in it's navigation options, but
+    // it exists in a stack with headerMode float, add a negative margin to
+    // compensate for the hidden header
```

Comment:
hmm, so the whole card is moved up, interesting. This is probably the most reasonable thing for now, but I wonder if we should instead make the header hight padding some explicit thing in the future, and in this case it would just be 0.

Reply:
yah, that's probably fair. Maybe this can be addressed in future changes to Stack.

Reply:
I'm seeing some flickers sometimes on app load where a screen without header is offset by the header height, not 100% sure but it might be caused by this because we measure the header view, I think we'd have to use the same value that's used to add the header height padding like @ericvicenti mentionned.

Reply:
I can confirm this is the issue, using state does help a bit but does not solve the issue. The only solution is to have the header height synchronously. It is exposed as a constant on the Header component but this will break custom header components if we use that.